content/en/docs/reference/command-line-tools-reference/kubelet.md

```diff
@@ -1024,6 +1024,13 @@ kubelet [flags]
       <td></td><td style="line-height: 130%; word-wrap: break-word;">If > 0, limit registry pull QPS to this value.  If 0, unlimited. (default 5) (DEPRECATED: This parameter should be set via the config file specified by the Kubelet's --config flag. See https://kubernetes.io/docs/tasks/administer-cluster/kubelet-config-file/ for more information.)</td>
     </tr>
 
+     <tr>
+        <td colspan="2">--reserved-cpus string</td>
+     </tr>
```

Comment:
Hello @jianzzha . You should make this change in the kubelet cmd line code (k8s/k8s).

Reply:
@kbhawkey thanks,  this file is reverted.